kubetest/BUILD.bazel

```diff
@@ -39,9 +39,9 @@ go_library(
         "@com_github_aws_aws_sdk_go//service/ec2:go_default_library",
         "@com_github_azure_azure_sdk_for_go//services/resources/mgmt/2018-05-01/resources:go_default_library",
         "@com_github_azure_azure_storage_blob_go//azblob:go_default_library",
-        "@com_github_azure_go_autorest//autorest:go_default_library",
-        "@com_github_azure_go_autorest//autorest/adal:go_default_library",
-        "@com_github_azure_go_autorest//autorest/azure:go_default_library",
+        "@com_github_azure_go_autorest_autorest//:go_default_library",
```

Comment:
Is this expected?

Reply:
See other comments on changes to go.mod after reworking this PR 😄